### PR TITLE
Daysgoby lulu

### PR DESCRIPTION
**Pull request by kdb424**
Picking up for @daysgobye since he can't find the "Allow maintainers to edit this pull request" button.

### CODE REVIEW

File: boards/boardsource/Lulu/kb.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+from kmk.kmk_keyboard import KMKKeyboard as _KMKKeyboard
+from kmk.scanners import DiodeOrientation
+from kmk.scanners.encoder import RotaryioEncoder
+from kmk.scanners.keypad import MatrixScanner
+
+
+class KMKKeyboard(_KMKKeyboard):
+    def __init__(self):
+        # create and register the scanner
+        self.matrix =[MatrixScanner(
```
Comment (klardotsh) [inline boards/boardsource/Lulu/kb.py:12]: Formatting here is funky in a way I suspect black should be complaining about (?)

drop this to a new line at the same indent level as its closing paren and the opening of `RotaryioEncoder`

File: boards/boardsource/Lulu/kb.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    # NOQA
+    # flake8: noqa
+    coord_mapping = [
+     0,  1,  2,  3,  4,  5,         37, 36, 35, 34, 33, 32,
+     6,  7,  8,  9, 10, 11,         43, 42, 41, 40, 39, 38,
+    12, 13, 14, 15, 16, 17,         49, 48, 47, 46, 45, 44,
+    18, 19, 20, 21, 22, 23, 29, 61, 55, 54, 53, 52, 51, 50,
+             25,26, 27, 28,         60, 59, 58, 57,
+             30,31,62,63
```
Comment (klardotsh) [inline boards/boardsource/Lulu/kb.py:69]: what are these? thumb keys? should these be further to the right visually?

File: boards/boardsource/Lulu/main.py [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+            0: OledReactionType.LAYER,
+            1: ['qwertyzzz', 'nums', 'shifted', 'leds', '5', '6', '7', '8'],
+        },
+    ),
+    toDisplay=OledDisplayMode.TXT,
+    flip=False,
+)
+keyboard.extensions.append(oled_ext)
+rgb_ext = Rgb_matrix(
+    ledDisplay=[
```
Comment (klardotsh) [inline boards/boardsource/Lulu/main.py:46]: Can this have a comment or be in a named variable describing what it is? this currently gives little explanation of what's going on

**Review by klardotsh — APPROVED** [review]

File: boards/boardsource/Lulu/kb.py (hunk 1 above)
Comment (kdb424) [reply boards/boardsource/Lulu/kb.py:12]: Black formatted that, so.... I hope not. Can do though.

File: boards/boardsource/Lulu/kb.py (hunk 2 above)
Comment (kdb424) [reply boards/boardsource/Lulu/kb.py:69]: Uhhhh, encoder I think? Not really sure honestly, but this looks way less bad that what I had.

File: boards/boardsource/Lulu/main.py (hunk 3 above)
Comment (kdb424) [reply boards/boardsource/Lulu/main.py:46]: @daysgobye Gonna need some info. I just did some formatting.

**Comment by kdb424** [comment]
Yolo merging. Will happily come back and fix whatever needs done if any complaints exist, but would rather have this board available for people that bought it.

**Comment by daysgobye** [comment]
thank you kdb!! sorry for my GitHub noobness. I will pr in the fixes [klardotsh](https://github.com/klardotsh) wants. ity was my bad for pulling it right from the peg maps.

**Comment by kdb424** [comment]
Think I handled most of them. Formatted with black, and hand formatted past that. Should be minimal changes to make it nicer if you so wish.

**Pull request merged**